src/__tests__/dates.test.ts

```diff
@@ -38,3 +38,21 @@ describe("UTCISODateFromString", () => {
     });
   });
 });
+
+describe("DateFromTimestamp", () => {
+  it("should validate a timestamp", async () => {
+    const timestamp = 1577836800000
+    const validation = DateFromTimestamp.decode(timestamp);
+    expect(isRight(validation)).toBeTruthy();
+    expect(DateFromTimestamp.is(new Date())).toBeTruthy();
```

Comment:
What if we check that the decoded type is a Date?
```suggestion
    expect(validation.value instanceof Date).toBeTruthy();
    expect(DateFromTimestamp.is(new Date())).toBeTruthy();
```